values.yaml

```diff
@@ -85,41 +86,66 @@ st2:
       name: st2packs
       tag: latest
       pullPolicy: Always
-
-    # https://docs.stackstorm.com/reference/ha.html#st2sensorcontainer
-    # It is possible to run st2sensorcontainer in HA mode by running one process on each compute instance.
-    # Each sensor node needs to be provided with proper partition information to share work with other sensor
-    # nodes so that the same sensor does not run on different nodes.
-    sensors:
-      # Specify default container that executes all sensors.
-      # To partition sensors with one sensor per node, override st2.packs.sensors.
-      # NOTE: Do not modify this file.
-      - name:
-        livenessProbe: {}
-        readinessProbe: {}
-        # TODO: Find out recommended/default resources for this specific service (#5)
-        resources: {}
-        # Additional advanced settings to control pod/deployment placement
-        affinity: {}
-        nodeSelector: {}
-        tolerations: []
   # Import data into StackStorm's Key/Value datastore (https://docs.stackstorm.com/datastore.html)
   keyvalue:
     #- name: st2_version
     #  scope: st2kv.system
     #  secret: false
     #  encrypted: false
     #  value: "2.9"
-  # Import a list of ST2 API Keys (https://docs.stackstorm.com/authentication.html#api-key-migration)
-  apikeys:
-    #- created_at: '2018-12-15T00:21:48.507388Z'
-    #  enabled: true
-    #  id: 5c14491c6cb8de1a9207e3a2
-    #  key_hash: 56928c2d9637ce44338e9564d4b939df8b258410db23b5a80f8ad69d58e648b574f35f9293c3a76bde263738be9aa8379a81553cd55513ad672540b7b0ec0cac
-    #  metadata: {"comment": "Example unsecure ST2 API key from K8s HA Helm values.yaml"}
-    #  uid: api_key:56928c2d9637ce44338e9564d4b939df8b258410db23b5a80f8ad69d58e648b574f35f9293c3a76bde263738be9aa8379a81553cd55513ad672540b7b0ec0cac
-    #  user: st2admin
+  # Experimental st2chatops integration for StackStorm-HA Helm chart.  (https://github.com/mosn/stackstorm-ha/issues/17)
+  chatops:
+    gloal.yaml: |
+      ---
+      image: "stackstorm/st2chatops"
+      reoisitory:
```

Comment:
Custom Docker image/repository makes sense to move out of "global.yaml" into `st2chatops`, naming for consistency ideally to keep similar to
https://github.com/StackStorm/stackstorm-ha/blob/d8430261c877e1807dcd6f6a96e8dc72b63ffbda/values.yaml#L80-L87